### PR TITLE
Add __init__.pyi files to git to help mypy when downloading from git

### PR DESCRIPTION
**Pull request by davfsa**
### Summary
Allow `mypy --strict` to work correctly when installing from Github

### Checklist

- [x] I have run `nox` and all the pipelines have passed.
- [x] I have made unittests according to the code I have added/modified/deleted.

### CODE REVIEW

File: pipelines/mypy.nox.py [hunk 1]
```diff
@@ -46,3 +46,13 @@ def generate_stubs(session: nox.Session) -> None:
...
+
+    for path in STUBGEN_GENERATE:
+        # We add "i" because we turn ".py" into ".pyi"
+        stub_path = path + "i"
+
+        with open(stub_path, "r") as fp:
+            content = fp.read()
+
+        with open(stub_path, "w") as fp:
+            fp.write("# This file was automatically generated by `nox -s generate-stubs`. Do not update!\n\n" + content)
```
Comment (FasterSpeeding) [inline pipelines/mypy.nox.py:58]: Probably better to say something like "do not manually update/modify" here

Comment (davfsa) [reply pipelines/mypy.nox.py]: Updated :+1:

Comment (FasterSpeeding) [reply pipelines/mypy.nox.py]: You forgot to re-run the nox task

**Review by FasterSpeeding — APPROVED** [review]

File: pipelines/mypy.nox.py (hunk 1 above)
Comment (davfsa) [reply pipelines/mypy.nox.py]: Yep :sweat_smile:

**Pull request merged**